website/src/components/repl/Tabs.tsx

```diff
@@ -0,0 +1,45 @@
+import { css } from "@emotion/css";
+import React from "react";
+
+export type TabsOptions = {
+  current: string;
+  labels: string[];
+  onClick: (current: string) => void;
+};
+
+export default function Tabs({ current, labels, onClick }: TabsOptions) {
+  return (
+    <div className={`padding--xs ${styles.container}`}>
```

Comment:
Could we use the pills component here?

https://infima.dev/docs/components/pills

Reply:
<img width="1301" height="368" alt="image" src="https://github.com/user-attachments/assets/0925e311-a36f-4003-b777-7d8179d3cfa7" />
<img width="1457" height="380" alt="image" src="https://github.com/user-attachments/assets/d844d6fb-2a9f-4b53-92cf-f05dcdc973ae" />
Both look good to me, the pills might need a border.